pkg/sdk/accounts_integration_test.go

```diff
@@ -63,6 +60,13 @@ func TestInt_AccountCreate(t *testing.T) {
 		}
 		err = client.Accounts.Create(ctx, accountID, opts)
 		require.NoError(t, err)
+
+		t.Logf("sleeping for 5 seconds to wait for account to be created")
```

Comment:
I have two remarks to that, one for now and the other for later:
1. this code is repeated, so please extract the sleep function with the seconds parameter and description parameter.
2. Sleeping is generally not the most excellent solution in tests; we are adding the fixed time, which makes our tests longer. A better solution is to use the await test pattern, which, in general, works like this:
-  there should be a function `awaitUntil` where we can specify timeout and condition check interval
- we use the function above providing a condition to check: in this case, fetch the account and check if there is no error
- method checks the condition several times using the check interval provided and ends with the first success; after the timeout and no successes it fails, and the test should be marked as failed
- this way, we do not wait these 5 seconds every time, but we wait 5 seconds at most

Let's introduce the second idea in a separate PR.

Reply:
i have used the github.com/avast/retry-go instead of writing a helper function. we were already using this library somewhere else. i guess it will work.